modules/@apostrophecms/ui/ui/apos/lib/i18next.js

```diff
@@ -33,6 +33,10 @@ export default {
     // multiple layers of code, as a single `label`
     // property for instance
     Vue.prototype.$t = (phrase, options) => {
+      if (phrase == null) {
```

Comment:
Should this check for strict equality? Maybe needs to check undefined and null?

Reply:
Alright. I thought this was against our linter rules.

Reply:
It's a common practice for a really useful test (nullishness), so the standard linter rules permit this specific one. Not == in general.